src/app/utils/BountyUtils.ts

```diff
@@ -199,6 +231,24 @@ const BountyUtils = {
                 title += '\n(Infinite claims available)';
             }
         }
+        if (bountyRecord.assign) {
```

Comment:
TODO: moving assign and gated info out from embed and into title

Reply:
Evergreen, application, assign, and gating all seemed like "types" of bounty, or properties of a bounty, rather than a state. The embeds were getting really busy mixing state history with these property-related things, and putting these in the title made it more clear how this bounty was going to act or require.